src/Development/IDE/Core/Rules.hs

```diff
@@ -504,7 +506,11 @@ getHiFileRule = defineEarlyCutoff $ \GetHiFile f -> do
               case r of
                 Right iface -> do
                   let result = HiFileResult ms iface
+#if MIN_GHC_API_VERSION(8,10,0)
```

Comment:
Please keep all CPP in the `Compat` module.

Reply:
Done